### DISCUSSION TITLE
Tablet registers every click has two clicks [XP pen Star G640s]

### DISCUSSION THREAD

**Discussion by Peasman** [OpenTabletDriver/OpenTabletDriver#1942] [Q&A] [answered]
After the latest update (6.0.1) my tablet only does double clicks. I tried using the Debounce plugin but it didn't work. Anyone have an Idea how this can happen?
Im using the XPPen G640 S with Devocubs antichatter and nothing else. 
Help is really appreciated!

**Comment by jamesbt365** [comment, marked as answer]
Fully kill OpenTabletDriver, then reopen it, it should fix the issue if not then help > export diagnostics and send that file here. (.txt preferred)

**Reply by Peasman** [reply]
This sadly did not help. Here are the diagnostics:
[diagnostics.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/7876780/diagnostics.txt)

**Reply by jamesbt365** [reply]
Should be patched now.

**Reply by Peasman** [reply]
Should this be fixed in 6.0.3? This issue is still present with the newest update.

**Comment by Peasman** [comment]
I just found that it's related to devocubs antichatter. When it is disabled its working normally.

**Reply by jamesbt365** [reply]
Prob fixed in an action build for it
